pkgs/applications/audio/sublime-music/default.nix

```diff
@@ -57,21 +57,17 @@ python3Packages.buildPythonApplication rec {
    ++ lib.optional serverSupport bottle
   ;
 
-  postPatch = ''
-    sed -i "/--cov/d" setup.cfg
-    sed -i "/--no-cov-on-fail/d" setup.cfg
-  '';
-
```

Comment:
Running coverage-related parts don't add value for us. Please revert the removal.